src/makeRobotAccountant.js

```diff
@@ -1,12 +1,27 @@
 'use strict';
-
 /**
  *
  * @return {function}
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = x => {
+    const add = y => {
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return x + y;
+    };
+
+    count++;
+
+    return add;
+  };
+
+  return getSum;
```

Comment:
BTW it can be simplify. But it's up to you 
```suggestion
  return x => {
   count++;
    return y => {
      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return x + y;
    };
  };
```